jpa/src/main/java/org/jboss/as/jpa/container/AbstractEntityManager.java

```diff
@@ -62,8 +62,6 @@
     public static final String PESSIMISTIC_WRITE_LOCK_MODE = "pessimistic_write";
     public static final String NONE_LOCK_MODE = "none";
 
-    private final transient boolean isTraceEnabled = ROOT_LOGGER.isTraceEnabled();
```

Comment:
Yes, its intended as a performance optimization, since the AbstractEntityManager methods are called very often by applications.  The scope of the objects varies (could be per transaction, per component invocation or stateful bean lifetime (xpc)).